### PR TITLE
Bump gradle wrapper to 7.4.1 + centralize lib version mgmt

### PR DESCRIPTION
**Pull request by alkum**
Update the gradle wrapper using

```
./gradlew wrapper --gradle-version 7.4.1 --distribution-type bin
```

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=b586e04868a22fd817c8971330fec37e298f3242eb85c374181b12d637f80302
```
Comment (chimp1984) [inline gradle/wrapper/gradle-wrapper.properties:3]: Is distributionSha256Sum not used anymore?

**Review by chimp1984 — COMMENTED** [review]
See open questions regarding `distributionSha256Sum`

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (alkum) [reply gradle/wrapper/gradle-wrapper.properties:3]: My mistake, fixed.

**Comment by alkum** [comment]
> See open questions regarding `distributionSha256Sum`

Fixed. Sorry was my bad, late at night and didnt realise the checksum was missing.

**Comment by alkum** [comment]
Related to https://github.com/bisq-network/bisq2/issues/178

**Review by chimp1984 — APPROVED** [review]
utACK

**Pull request merged**